lib/tasks/deployment/20190503121932_create_oxford_amr_config.rake

```diff
@@ -0,0 +1,24 @@
+namespace :after_party do
+  desc 'Deployment task: create_oxford_amr_config'
+  task create_oxford_amr_config: :environment do
+    puts "Running deploy task 'create_oxford_amr_config'"
+
+    reading_fields = '00:30,01:00,01:30,02:00,02:30,03:00,03:30,04:00,04:30,05:00,05:30,06:00,06:30,07:00,07:30,08:00,08:30,09:00,09:30,10:00,10:30,11:00,11:30,12:00,12:30,13:00,13:30,14:00,14:30,15:00,15:30,16:00,16:30,17:00,17:30,18:00,18:30,19:00,19:30,20:00,20:30,21:00,21:30,22:00,22:30,23:00,23:30,24:00'
```

Comment:
Not entirely sure whether the readings start at 00:00 or 00:30. Although, we don't have `handle_off_by_one` turned on for any configs so does it even matter? Claudia is going to investigate.

Reply:
Hmm, I thought it was Frome which should have ```handle off by one``` turned on so I'm a bit confused now!

Reply:
Ah, it should be on for Frome historical - no idea why it's off but we're not importing any historic data at the moment either...